tests/test_twittercard.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import json
+import unittest
+
+from extruct.twittercard import TwitterCardExtractor
+from tests import get_testdata, jsonize_dict
+
+class TestTwittercard(unittest.TestCase):
```

Comment:
Are these tests run? I don't see them in the log here https://github.com/scrapinghub/extruct/runs/6452050839?check_suite_focus=true#step:5:20